lib/aliquot-pay.rb

```diff
@@ -72,36 +72,63 @@ def self.four_byte_length(str)
     [str.length].pack('V')
   end
 
+  def self.generate_signature(*args)
+    args.map do |s|
+      four_byte_length(s) + s
+    end.join('')
```

Comment:
Space as separator is default.

Reply:
*Empty string*  as separator is default 🙂

Reply:
I'm sorry... I thought "empty string" but wrote "space"...